build-aux/m4/ax_boost_base.m4

```diff
@@ -1,5 +1,5 @@
 # ===========================================================================
```

Comment:
I verified that this file matches http://git.savannah.gnu.org/gitweb/?p=autoconf-archive.git;a=blob_plain;f=m4/ax_boost_base.m4 apart from the following trivial change:
```diff
13c13
< #   If no path to the installed boost library is given the macro searchs
---
> #   If no path to the installed boost library is given the macro searches
```